### ISSUE TITLE
Do we really need to always force a rebuild of a case when pelayout is changed?

### ISSUE THREAD

**Issue by jgfouca** [ESMCI/cime#4551]
It seems to me and @golaz , that as long as threads vs. unthreaded is not changed, a rebuild should not be necessary.

**Comment by rljacob** [comment]
You actually don't need to do a full rebuild, despite what the instructions say.   Do the "./case_setup --reset" and then do "./case.build" and skip the clean.

**Comment by jedwards4b** [comment]
This is a legacy requirement of pop and cice5.   I believe we can remove this warning now.

**Comment by jgfouca** [comment]
@rljacob , I should have been more clear. I think users want to make it so CIME does not think the case needs to be rebuilt when a minor pelayout change is made. So:

```
./xmlchange *change peylaout*
./case.setup
./case.submit
```

@golaz, does that cover your use cases?

**Comment by rljacob** [comment]
I agree that's the right way to do this.  I was pointing out a workaround to doing a complete rebuild.

**Comment by jedwards4b** [comment]
At least for cesm the case.build step is very fast for this case, it simply assures that the executable is up to date.  I think we should leave it as a requirement.
